index.html

```diff
@@ -19,7 +19,14 @@ <h1 id="title">Survey Form</h1>
         <label for="email" id="email-label">Email</label>
         <input type="email" id="email" placeholder="Your Email" required />
```

Comment:
- [ ] You should actually checks the code to look for errors and bugs and leaves nice comments containing change suggestions for each. Examples of issues you may look out for when performing the review:

* Lack of [semantic HTML tags use](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
* Not applying [HTML & CSS best practices correclty](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
* Bugs missed the first time the project was built.
* etc

Reply:
- [ ] As mentioned by the previous reviewer, the goal of this project is to check for any bugs or possible improvements and implement the GitHub inline comments feature to point this out in a friendly and suggestions way :bulb: :heavy_check_mark:. Please make sure to leave at least 2 inline comments as stated in the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#:~:text=Add%20as%20many%20comments%20as%20possible%20in%20the%20given%20time%20and%20finish%20your%20review.%20Remember%20to%20add%20a%20minimum%20of%202%20comments%20%2D%20you%20will%20definitely%20find%20something%20that%20could%20have%20been%20done%20better!) 

In this lesson, you can find more information and tips about the [GitHub review features](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) and how to perform a code review :memo: :100: